core/src/br/cefetmg/games/screens/OverworldScreen.java

```diff
@@ -661,6 +769,33 @@ public void update(float dt) {
         } else {
             hideStage(stage5);
         }
-        stage.act(dt);
+        
+        
+        if(isVictory) {
+            backgroundMusic.stop();
+            victoryMusic.play();
+            counter++;
+            if(counter>MAX_TIME_TO_CHANGE_SPRITESHEET) {
+                victoryCatSpritesheetNumber++;
+                counter=0;
+            }
+            if(victoryCatSpritesheetNumber>NUM_SPRITESHEETS-1) {
+                victoryCatSpritesheetNumber = 0;
+                victoryMusic.stop();
+                backgroundMusic.play();
+                isVictory = false;
+            }
+            for(int i=0; i<13; i++) {
+                catVictoryMovingR[i].animationTime += dt;
+                catVictoryMovingB[i].animationTime += dt;
+            }
+        }
+        if(score>=POINTS_TO_VICTORY_ANIMATION && !victoryPlayed) {
+            isVictory = true;
+            victoryPlayed = true;
+        }
+        System.out.println("isVictory: "+isVictory);
```

Comment:
:((((

Reply:
Pela carinha triste percebi as merdas que eu fiz no código. Tirei os prints que eu estava usando pra debug e tirei o for totalmente desnecessário. Meu cérebro já tá fritando. Estou só o bagaço nesse final de semestre. Me desculpe professor.